frontend/apps/web/app/(mgmt)/[account]/new/transformer/page.tsx

```diff
@@ -180,7 +182,17 @@ export default function NewTransformer(): ReactElement {
               <FormItem>
                 <FormLabel>Source Transformer</FormLabel>
                 <FormDescription>
-                  The system transformer to clone.
+                  The system transformer to clone.{' '}
+                  {formSource !== 0 &&
+                    formSource !== null &&
+                    TransformerSource[formSource ?? 0] !=
+                      'GENERATE_JAVASCRIPT' &&
+                    TransformerSource[formSource ?? 0] !=
+                      'TRANSFORM_JAVASCRIPT' && (
```

Comment:
There is a function that can be called  in this file (`getTransformerSource`) that will convert the number to `TransformerSource`. That way you can use typesafe values.

Reply:
Also, why is ti being omitted for the javascript ones? Is there not a doc page for those?

Reply:
Javascript ones have their own special docs link that points to the custom transformers page which is a guide vs. these docs link go to the system transformers. I was originally thinking of leaving it there since the learn more link was right next to the code input instead of having to scroll up to the top of the page. But i can update the `constructDocLink` function to handle that case.